### ISSUE TITLE
Turing Machine

### ISSUE THREAD

**Issue by Kolibrix2** [TomWhitwell/TuringMachine#189]
Hello I hope you are well. Im buying the kit from the tuning machine last week and now I build it full finish but when I go frame the trigger ( midi cv trigger from Behringer) The Turing machine does nothing ( I go frome trig into clock and then from out into trig of the synt ) if I start the model only 2 LED are one. Kann every one help me to fix that problem ?

**Comment by Kolibrix2** [comment]
Good morning I have an update when I connect CV out to Pulse this LED lights up but when I connect it to the clock nothing happens generally does not come out of the out at the top only the two left LEDs and the very right one can possibly someone help thanks in advance

**Comment by Thonk-Steve** [comment]
Hi, could you maybe upload a video of what you are describing?

**Comment by Kolibrix2** [comment]
I don't know how to go about troubleshooting ( it's my first DIY module ) Here is the video I hope you can help me :)
I hope a YouTube Link its okay : https://www.youtube.com/shorts/lD_nOBUHOGA

**Comment by Thonk-Steve** [comment]
Thanks, what is the jack you are patching into on the Behringer called?

**Comment by Kolibrix2** [comment]
I tried it with CV out and Gate out of the TD3 with my Behringer Midi CV I tried the outputs CV and Trig (if I put it directly into my voice it works too)

**Comment by Thonk-Steve** [comment]
Yes the Pulse jack on the Turing is an OUTPUT, not input. You won't have
damaged anything doing this I don't think though.

**Comment by Kolibrix2** [comment]
oh ok that was just a desperate attempt to figure out if I'm just hooking it up wrong :/ now how can I figure out why when I put a CV signal into the clock the out doesn't get anything and the lights don't start blinking?

**Comment by Thonk-Steve** [comment]
You should be patching the sync out or clock out on the Behringer to the
clock input on the Turing (not the Behringer CV out as you do in the
video), then the output of the Turing to the 1v/oct input of an oscillator.

**Comment by Kolibrix2** [comment]
So I can't trigger the module via the trig out ore CV out of my midi CV from Behringer ?( sorry I'm absolutely new in this world :D )
![IMG_1628](https://user-images.githubusercontent.com/119662258/206916887-5f5dcd6b-e8c5-4839-8e60-f7494ff05c63.JPG)

**Comment by Kolibrix2** [comment]
I want to sync the whole thing with my Octatrack, so I bought the CV to midi module

**Comment by Kolibrix2** [comment]
maybe I need this one ?[ ](https://www.thonk.co.uk/shop/startup/)

**Comment by Kolibrix2** [comment]
maybe I need this one ? https://www.thonk.co.uk/shop/startup/

**Comment by Thonk-Steve** [comment]
> So I can't trigger the module via the trig out ore CV out of my midi CV
from Behringer ?( sorry I'm absolutely new in this world :D )

No CV out will be sending the note CVS, i.e. the melody it's playing, so
search your Behringer manual for sending the clock out or sync out, i'm not
familiar with that device sorry.

**Comment by Kolibrix2** [comment]
Hello, today I tried it with a doepfer a190-8 and unfortunately it didn't work either, maybe someone gave me a tip ? 
![image](https://user-images.githubusercontent.com/119662258/208074153-6136f980-1a2c-4c50-9c32-8aa229bd371a.jpg)
![image](https://user-images.githubusercontent.com/119662258/208074273-2def575a-323a-4520-bbdb-bdb41936c544.jpg)
![image](https://user-images.githubusercontent.com/119662258/208074585-3a865f21-92d3-40be-b040-71ca09e359dc.jpg)
![image](https://user-images.githubusercontent.com/119662258/208074666-02659ded-2eb8-4cdb-aa22-192dc014448e.jpg)
Here are a few pictures, maybe someone recognizes yes, because mistakes, thank you

**Comment by TomWhitwell** [comment]
Hi, I'm not certain, but unfortunately it looks like this chip is the wrong way round. Pin 1 is top left, and pin 1 may be marked by that flattened circle.  
<img width="578" alt="image" src="https://user-images.githubusercontent.com/1890544/208091940-9a557860-4f10-4fee-95a8-5a7d92384742.png">
If your problem is you have no LEDs at all, and no response to the clock out of the Doepfer, that chip wouldn't make the difference, but it could possibly be scrambling / shorting power in the whole module (or have already damaged other chips). Try removing that chip to see if it makes any difference - the looping won't work, but you might at least be able to see pulses with clock input.

**Comment by Kolibrix2** [comment]
Thanks for everything, I installed the chip correctly and now it goes out when I go over the pulse but unfortunately nothing comes out of the out output, what could be the reason?

**Comment by Kolibrix2** [comment]
Okaay Thanks vor everting
